spring-cql/src/test/java/org/springframework/cassandra/config/CassandraCqlSessionFactoryBeanUnitTests.java

```diff
@@ -0,0 +1,349 @@
+/*
+ *  Copyright 2013-2016 the original author or authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cassandra.config;
+
+import static org.hamcrest.Matchers.equalTo;
```

Comment:
Will do.

Personally, I think it was a mistake for Java to ever allow wildcard imports.  It is always better to be explicit about what exactly is being imported (a dependency, used by a class), static or not.  It is a simple matter to include (auto) import classes from within an IDE so using wildcards has less value today then it did in the past.  Using wildcards also makes it more difficult, especially outside an IDE to determine where a conflict originates.  No where is this more apparent then with Mockito and Hamcrest Matchers.